schema.sql

```diff
@@ -5,5 +5,5 @@ CREATE TABLE animals(
     date_of_birth DATE,
     escape_attempts INT,
     neutered BOOLEAN,
-    weight_kg DECIMAL
-);
+    weight_kg DECIMAL,
+    species VARCHAR(40));
```

Comment:
- [ ] <b>OPTIONAL:</b> As mentioned by the previous reviewer, The best way to add new column in table is by using the `ALTER` command. because by doing so, you won't have to reset and re-create your table. EX: 

```sql 
ALTER TABLE animals ADD species varchar(255); 
```